locales/ms.json

```diff
@@ -105,7 +113,6 @@
         "addClinic": "mendaftar klinik",
         "adminDashboard": "Papan pemuka Admin",
         "github": "GitHub",
-        "home": "Laman rumah",
```

Comment:
Please make sure to merge with the latest main and not clobber new translations 🙇🏻‍♀️

Reply:
Agh, I thought I had pulled in the latest changes...

Reply:
@ann-kilzer 
I think when I ran this through `jq` it automatically cut out the extra `home` value since it's not valid json... could you take a look at this: https://github.com/ourjapanlife/findadoc-localization/blob/main/locales/ms.json#L108

Reply:
Ahh I see that now.

@samanthawxlow could you confirm which is the correct value for the Home page translation?

Reply:
I'm gonna guess that Laman rumah is the correct translation.

Could you go ahead and fix the merge conflicts

Reply:
@ann-kilzer 
Fixed the merge conflicts!